<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chk_1">
        <property name="text">
         <string>CehckBox 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_2">
        <property name="text">
         <string>CheckBox 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_3">
        <property name="text">
         <string>CheckBox 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_4">
        <property name="text">
         <string>CheckBox 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="groupchk_1">
          <property name="text">
           <string>GroupCheck 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="groupchk_2">
          <property name="text">
           <string>GroupCheck 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="groupchk_3">
          <property name="text">
           <string>GroupCheck 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="groupchk_4">
          <property name="text">
           <string>GroupCheck 4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
